Formula/speex.rb

```diff
@@ -13,11 +13,19 @@ class Speex < Formula
 
   depends_on "pkg-config" => :build
   depends_on "libogg" => :recommended
+  depends_on "speexdsp" => :optional
 
   def install
     ENV.deparallelize
-    system "./configure", "--disable-debug", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
+    ENV["SPEEXDSP_CFLAGS"] = "-I#{HOMEBREW_PREFIX}/include/speex" if build.with? "speexdsp"
```

Comment:
And if it doesn't, use `#{Formula["speex"].opt_include}"` and `#{Formula["speex"].opt_lib}"` instead of using `HOMEBREW_PREFIX`.